### DISCUSSION TITLE
Code level in the Main is different

### DISCUSSION THREAD

**Discussion by Daedalin** [community/community#42881] [Codespaces]
### Select Topic Area

Bug

### Body

Today when I wanted to make a pull request I noticed that there is a difference in 4 files. Which should not be present.

If you look at the history of one of the files you see only one commit from Sep 11, 2022,
![image](https://user-images.githubusercontent.com/57679443/210003394-14c03b77-51fc-44c9-a002-ea2c358bb8fa.png)

but if you look at the general history you see 3 commits on that day.
![image](https://user-images.githubusercontent.com/57679443/210003604-7e6d96fa-26c3-4646-be41-09852a54b64b.png)
![image](https://user-images.githubusercontent.com/57679443/210004044-583cd05e-a7dd-4938-b3be-ab991d7bb974.png)

**Comment by cmbrose** [comment]
👋 Hi @Daedalin - this isn't really a question for Codespaces, you will be able to get better help if you recategorize this discussion to Pull Requests or Repositories. 

When you look at the history of a single file, it doesn't show all the commits in the entire repository, only the commits that affected that file. That may explain what you are seeing.
